modules/AlphaWalletFoundation/AlphaWalletFoundation/Transactions/EtherscanSingleChainTransactionProvider.swift

```diff
@@ -184,7 +183,7 @@ class EtherscanSingleChainTransactionProvider: SingleChainTransactionProvider {
 
         transactionsNetworkProvider
             .getTransactions(startBlock: 1, endBlock: oldestCachedTransaction.blockNumber - 1, sortOrder: .desc)
-            .sink(receiveCompletion: { [transactionsTracker] result in
+            .sinkAsync(receiveCompletion: { [transactionsTracker] result in
```

Comment:
Wouldn't it be the same?

Reply:
there is an issue with accessing value type types like arrays, sets, dictionaries in background, updating it with `sinkAsync` avoids this issue as we don't store it in any array, and just release subscription when receive completion

Reply:
Ah I see.